vscode-extension/src/extension.ts

```diff
@@ -42,6 +123,242 @@ export function deactivate() {
   console.log("Deactivated AIConfig extension");
 }
 
+/**
+ * Creates a new AIConfig file in the editor.
+ */
+async function createNewAIConfig(
+  context: vscode.ExtensionContext,
+  aiconfigEditorManager: AIConfigEditorManager,
+  mode: "json" | "yaml" = "json"
+) {
+  const workspaceUri = vscode.workspace.workspaceFolders
+    ? vscode.workspace.workspaceFolders[0].uri
+    : null;
+  const untitledUri = workspaceUri
+    ? workspaceUri.with({
+        scheme: "untitled",
+        path: `${workspaceUri.path}/untitled.aiconfig.${mode}`,
+      })
+    : vscode.Uri.parse(`untitled:untitled.aiconfig.${mode}`);
+
+  // Specify the initial content here
+  const newAIConfigJSON = vscode.Uri.joinPath(
```

Comment:
Should this use untitledUri instead? I see the only other line using untitledUri below is commented out